assertions.js

```diff
@@ -1,43 +1,38 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-class Assert {
-  isNumber(value, name) {
-    if(typeof value !== 'number') {
-      throw new TypeError(`"${name}" must be number.`);
-    }
+exports.isNumber = (value, name) => {
```

Comment:
This is CommonJS style.  I think we could use full ES6 module [export](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export) style:
```js
export function isNumber(...) { ... }
```

Reply:
Good catch! Thank you for pointing that out. Updated it to es6 style.